src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  static updateAliveList = () => {
+    Animal.alive = Animal.alive.filter(animal => animal.health > 0)
+  }
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
```

Comment:
- "Health is 100 by default." So it means that you still can pass a health parameter to the constructor if needed
- 100 is a magic value, create a constant for it
- Put constructor before methods